static/templates/settings/api_key_modal.hbs

```diff
@@ -1,42 +1,43 @@
+        <div class="modal__container" role="dialog" aria-modal="true" aria-labelledby="api_key_modal_label">
+            <header class="modal__header">
+                <h1 class="modal__title" id="api_key_modal_label">
+                    {{t "Show API key" }}
+                </h1>
+                <button class="modal__close" aria-label="{{t 'Close modal' }}" data-micromodal-close></button>
+            </header>
+            <main class="modal__content">
+                <div id="password_confirmation">
+                    <div id="api_key_form">
+                        <p>{{t "Please re-enter your password to confirm your identity." }}
+                            <a href="/accounts/password/reset/" target="_blank" rel="noopener noreferrer">
+                                {{t "Never had one? Forgotten it?" }}
+                            </a>
+                        </p>
+                        <div class="password-div">
+                            <label for="password" class="control-label">{{t "Current password" }}</label>
+                            <input type="password" autocomplete="off" name="password" id="get_api_key_password" value="" />
+                            <i class="fa fa-eye-slash password_visibility_toggle tippy-zulip-tooltip" role="button"></i>
+                        </div>
+                    </div>
+                    <span class="alert-notification" id="api_key_status"></span>
+                </div>
+                <div id="show_api_key">
+                    <p>{{t "Your API key:" }}</p>
+                    <p><b><span id="api_key_value"></span></b></p>
+                    <div id="user_api_key_error" class="text-error"></div>
                 </div>
-                <button type="submit" name="view_api_key" id="get_api_key_button" class="button sea-green">
+            </main>
+            <footer class="modal__footer">
+                <button type="submit" name="view_api_key" id="get_api_key_button" class="modal__btn dialog_submit_button">
                     {{t 'Get API key' }}
                 </button>
-            </form>
-        </div>
-        <div id="show_api_key">
-            <p>{{t "Your API key:" }}</p>
-            <p><b><span id="api_key_value"></span></b></p>
-            <div id="api_key_buttons">
-                <button type="submit" class="button white rounded regenerate_api_key">
-                    {{t "Generate new API key" }}
-                </button>
-                <a id="download_zuliprc" download="{{zuliprc}}" class="button sea-green">
-                    {{t "Download .zuliprc" }}
-                </a>
-            </div>
-            <div id="user_api_key_error" class="text-error"></div>
+                <div id="api_key_buttons">
+                    <button class="modal__btn dialog_submit_button" id="regenerate_api_key" aria-label="{{t 'Close this dialog window' }}">{{t "Generate new API key" }}</button>
```

Comment:
The `aria-label` attribute is incorrect.

Reply:
Fixed